packages/common/src/themes/codesandbox-black.js

```diff
@@ -206,9 +206,8 @@ const colors = {
   },
   sideBar: {
     background: tokens.grays[700],
-    hoverBackground: tokens.green,
     border: tokens.grays[500],
-    foreground: tokens.grays[300],
+    foreground: tokens.grays[200],
```

Comment:
⚠️ Theme change! The design has 200 not 300 🤔 this seems like a bug that was introduced a while back

Reply:
Good fix!